src/main/java/bhs/devilbotz/subsystems/DriveTrain.java

```diff
@@ -80,7 +84,7 @@ public class DriveTrain extends SubsystemBase {
           Robot.getSysIdConstant("FEED_FORWARD_LINEAR_A").asDouble());
 
   // Defines the field, which is used to display the robot's position on the field in Shuffleboard.
-  private final Field2d field = new Field2d();
+  public final Field2d field = new Field2d();
```

Comment:
Hi Maciej, this was made public, but doesn't seem to be accessed by anyone else.

Reply:
whoops.. probably left over from hacking I did to figure out how to get this working, will put it back way was.